src/ring_theory/valuation/valuation_ring.lean

```diff
@@ -270,6 +270,83 @@ end
 
 section
 
+variables (R : Type*) [comm_ring R] [is_domain R] {K : Type*}
```

Comment:
Let's make `R` implicit for these iffs, since it appears on both sides.
```suggestion
variables {R : Type*} [comm_ring R] [is_domain R] {K : Type*}
```